### DISCUSSION TITLE
The Parthenon was built in honor of which 🇬🇷 goddess?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#807] [Galaxy Trivia] [answered]
**Bonus:** How many copies of Parthenon can you find in the US?

**Comment by peterbaccaro** [comment, marked as answer]
The Parthenon, an ancient temple located on the Acropolis of Athens in Greece, was built in honor of the goddess **Athena**. Athena was the Greek goddess of wisdom, courage, inspiration, civilization, law and justice, strategic warfare, mathematics, strength, strategy, the arts, crafts, and skill.

There are **4** copies of Parthenon in the United States:

1. Second Bank of the United States, Philadelphia
2. Federal Hall National Memorial, New York
3. Lincoln Memorial, Washington DC
4. Nashville’s Parthenon, Tennessee

![image](https://github.com/ParanoidUser/yolo/assets/74666079/bc8ad908-c338-4886-ad44-03086f0eafc7)

**Comment by SANKARAMDAS** [comment]
The Parthenon is a former temple on the Athenian Acropolis, Greece, that was dedicated to the goddess Athena during the fifth century BC.
https://en.wikipedia.org/wiki/Parthenon

3 Spectacular Replicas Of The Parthenon

Lincoln Memorial, Washington DC- built-in 1922 
Federal Hall National Memorial, New York - built in 1842
Second Bank of the United States, Philadelphia -built in 1824 by architect William Strickland
https://www.thecollector.com/replicas-parthenon/
